Loop/Managers/Alerts/DeviceAlertManager.swift

```diff
@@ -246,6 +230,61 @@ extension DeviceAlertManager {
 
 }
 
+// MARK: Alert storage access
+extension DeviceAlertManager {
+    
+    func getStoredEntries(startDate: Date, completion: @escaping (_ report: String) -> Void) {
+        alertStore.executeAlertQuery(since: startDate, limit: 100) {
+            switch $0 {
+            case .failure(let error): completion("Error: \(error)")
```

Comment:
👍 In this case, good point, you are right: this should be on a separate line, but (again, IMHO) because the other `case` is not a single line.  When all of the `case` statements are short, I very much prefer this:
```swift
switch number {
    case .one: return 1
    case .two: return 2
    case .three: return 3
}
```
over this:
```swift
switch number {
    case .one: 
        return 1
    case .two: 
        return 2
    case .three: 
        return 3
}
```
Again, just MHO.